_data/3.11-manual.yml

```diff
@@ -1,186 +1,145 @@
-      href: data-model-and-concepts.html
-      children:
-        - text: Databases
-          href: data-modeling-databases.html
-          children:
-            - text: Database Methods
-              href: data-modeling-databases-working-with.html
-        - text: Collections
-          href: data-modeling-collections.html
-          children:
-            - text: Collection Methods
-              href: data-modeling-collections-collection-methods.html
-            - text: Database Methods
-              href: data-modeling-collections-database-methods.html
-        - text: Documents
-          href: data-modeling-documents.html
-          children:
-            - text: Collection Methods
-              href: data-modeling-documents-document-methods.html
-            - text: Database Methods
-              href: data-modeling-documents-database-methods.html
-            - text: Schema Validation
-              href: data-modeling-documents-schema-validation.html
-            - text: Computed Values
-              href: data-modeling-documents-computed-values.html
-        - text: Views
-          href: data-modeling-views.html
-          children:
-            - text: Database Methods
-              href: data-modeling-views-database-methods.html
-            - text: View Methods
-              href: data-modeling-views-view-methods.html
-    - text: How to Interact with ArangoDB
-      href: how-to-interact-with-arangodb.html
-    - text: Setting up a cloud instance
-      href: quick-start-in-the-cloud.html
-    - text: On-premises installation
-      href: quick-start-on-premises.html
-    - text: Start using AQL
-      href: start-using-aql.html
-- subtitle: CORE TOPICS
-- text: Programs & Tools
-  href: programs.html
+    - text: Databases
+      href: data-modeling-databases.html
+      children:
+        - text: Database Methods
+          href: data-modeling-databases-working-with.html
+    - text: Collections
+      href: data-modeling-collections.html
+      children:
+        - text: Collection Methods
+          href: data-modeling-collections-collection-methods.html
+        - text: Database Methods
+          href: data-modeling-collections-database-methods.html
+    - text: Documents
+      href: data-modeling-documents.html
+      children:
+        - text: Collection Methods
+          href: data-modeling-documents-document-methods.html
+        - text: Database Methods
+          href: data-modeling-documents-database-methods.html
+        - text: Schema Validation
+          href: data-modeling-documents-schema-validation.html
+        - text: Computed Values
+          href: data-modeling-documents-computed-values.html
+    - text: Views
+      href: data-modeling-views.html
+      children:
+        - text: Database Methods
+          href: data-modeling-views-database-methods.html
+        - text: View Methods
+          href: data-modeling-views-view-methods.html
+- text: Operational Factors
```

Comment:
This is actually about data modeling (as in architecting solutions), whereas the above pages are about the data model of ArangoDB. We had both mixed together for a very long time and eventually separated the topics because they are quite different. Now, they are mixed together again. Is this really what we want?

Reply:
We need to find better names for these two, more distinguishable. How about something like:
- Data Model & Concepts -> How data is structured / Data structure in ArangoDB
- Data Modeling & Operational Factors > Model application data for ArangoDB

Agree that we should keep them separate. The question is where to place the second one (that's why it ended up here)?

Reply:
"Data model" and "Data modeling" are indeed too similar. Maybe something along the line of _planning_ and _designing_ solutions with ArangoDB? It could become the place for how to model application data and architecting solutions, although it would currently be pretty empty as we don't have much content.